System.Containers.Tasks/build/System.Containers.Tasks.targets

```diff
@@ -2,8 +2,10 @@
     <Target Name="ComputeContainerConfig">
         <!-- Compute private defaults -->
         <PropertyGroup Condition="$(ContainerBaseImage) == ''">
+            <_IsSelfContained Condition="'$(SelfContained)' == 'true' or '$(PublishSelfContained)' == 'true'">true</_IsSelfContained>
             <_ContainerBaseRegistry>https://mcr.microsoft.com</_ContainerBaseRegistry>
-            <_ContainerBaseImageName Condition="@(ProjectCapability->AnyHaveMetadataValue('Identity', 'AspNetCore'))">dotnet/aspnet</_ContainerBaseImageName>
+            <_ContainerBaseImageName Condition="'$(_IsSelfContained)' == 'true'">dotnet/runtime-deps</_ContainerBaseImageName>
```

Comment:
Ridiculous suggestion: Should we add a condition for `_ContainerBaseImageName == ''`? This would allow a customer to go self contained and specify the condition.

_Actually_ (leaving this up because why not): They can just set `ContainerBaseImage`. Ignore this one.